x/pairing/keeper/pairing.go

```diff
@@ -249,7 +249,10 @@ func (k Keeper) CalculateEffectiveAllowedCuPerEpochFromPolicies(policies []*plan
 	var policyTotalCuLimit []uint64
 	for _, policy := range policies {
 		if policy != nil {
-			if policy.EpochCuLimit == 0 || policy.TotalCuLimit == 0 {
+			if policy.EpochCuLimit == 0 {
```

Comment:
if policy.EpochCuLimit != 0
   policyEpochCuLimit = append(policyEpochCuLimit, policy.GetEpochCuLimit())

the same for the other limit